source/publishing_data/05_processing_data/processors/expression.rst

```diff
@@ -193,6 +195,11 @@ The last part of the ternary operator is optional, which means that the followin
 - ``'true' ? 'hello'`` returns ``'hello'``
 - ``'false' ? 'hello'`` returns an empty result
 
+.. admonition:: Note
+   :class: note
+
+   Some operations can fail if they involve a field which type was wrongly guessed by the platform at the creation of the dataset, even if the field type was modified afterward.
```

Comment:
There is a workaround using another processor : placing the **cast (Force type) processor** before the expression processor to force the type to text fixes the issue. Maybe it would be cool to add it to the note

Reply:
I think that we can suggest them to contact the support if they get stuck ?

Reply:
I'm not too sure about mentioning a workaround that uses a processor that is not available by default and that isn't documented. I think I'll just indicate that users should contact the support if they believe they are stuck because of this bug.